pallets/funding/src/functions.rs

```diff
@@ -1064,6 +1072,20 @@ impl<T: Config> Pallet<T> {
 			InvestorType::Professional => project_metadata.bidding_ticket_sizes.professional,
 			_ => return Err(Error::<T>::NotAllowed.into()),
 		};
+		let max_multiplier = match investor_type {
+			InvestorType::Retail => {
+				RetailParticipations::<T>::mutate(&did, |project_participations| {
+					if project_participations.contains(&project_id).not() {
+						// We don't care if it fails, since it means the user already has access to the max multiplier
+						let _ = project_participations.try_push(project_id);
+					}
+					retail_max_multiplier_for_participations(project_participations.len() as u8)
+				})
+			},
+
```

Comment:
Retail cannot bid, right? So why do we have a check on max_multiplier here? We return with error two lines above this statement if type is Retail

Reply:
Good point lol